src/test/javascript/spec/component/plagiarism/text-submission-viewer.component.spec.ts

```diff
@@ -33,7 +34,8 @@ describe('Text Submission Viewer Component', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, ArtemisPlagiarismModule, TranslateTestingModule],
+            imports: [ArtemisTestModule, TranslateTestingModule],
+            declarations: [TextSubmissionViewerComponent, MockComponent(SplitPaneHeaderComponent)],
```

Comment:
```suggestion
            imports: [ArtemisTestModule],
            declarations: [TextSubmissionViewerComponent, MockComponent(SplitPaneHeaderComponent), MockPipe(ArtemisTranslatePipe)],
```
I think you can remove the import if you mock the translation pipe.

Reply:
Thanks for the hint, done in bed923d